### ISSUE TITLE
Header normalize function leaves req.http.Host empty

### ISSUE THREAD

**Issue by jesperweiland** [mattiasgeniar/varnish-6.0-configuration-templates#2]
This function leaves the req.http.Host empty in when there's no Host header present. I saw this when monitoring an ip with Nagios check_http -I $ip

```
set req.http.Host = regsub(req.http.Host, ":[0-9]+", "");
```

I fixed it by checking if req.http.Host is not empty, but it should probably be the default in these templates.

```
if (req.http.Host) {
 set req.http.Host = regsub(req.http.Host, ":[0-9]+", "");
}
```

**Comment by emansom** [comment]
You might want to reconsider your service health checking tactics.

**Comment by mattiasgeniar** [comment]
This got fixed with 13c813c6bd83233596362447915fdb744a7ee0b5
